app/templates/administrative-units/administrative-unit/governing-bodies/governing-body/mandatory/new.hbs

```diff
@@ -0,0 +1,236 @@
+                <:label>Postcode</:label>
+                <:content>
+                  <AuInput
+                    @value={{@model.address.postcode}}
+                    @width="block"
+                    id="address-postal-code"
+                  />
+                </:content>
+              </Item>
+              <Item @labelFor="address-municipality">
+                <:label>Gemeente</:label>
+                <:content>
+                  <MunicipalitySelect
+                    @selected={{@model.address.municipality}}
+                    @onChange={{fn (mut @model.address.municipality)}}
+                  />
+                </:content>
+              </Item>
+              <Item @labelFor="address-province">
+                <:label>Provincie</:label>
+                <:content>
+                  <ProvinceSelect
+                    @selected={{@model.address.province}}
+                    @onChange={{fn (mut @model.address.province)}}
+                  />
+                </:content>
+              </Item>
+            </:left>
+            <:right as |Item|>
+              <Item @labelFor="contact-email">
+                <:label>E-mailadres</:label>
+                <:content>
+                  <AuInput
+                    @value={{@model.contact.email}}
+                    @width="block"
+                    id="contact-email"
+                    type="email"
+                  />
+                </:content>
+              </Item>
+              <Item @labelFor="contact-landline">
+                <:label>Telefoonnummer</:label>
+                <:content>
+                  <AuInput
+                    @value={{@model.contact.telephone}}
+                    @width="block"
+                    id="contact-landline" type="tel"
+                  />
+                </:content>
+              </Item>
+              <Item @labelFor="contact-mobile">
+                <:label>GSM nummer</:label>
+                <:content>
+                  <AuInput
+                    @width="block"
+                    @value={{@model.contactMobile.telephone}}
+                    id="contact-mobile"
+                    type="tel"
+                  />
+                </:content>
+              </Item>
+            </:right>
+          </Card.Columns>
+        </:card>
+      </EditCard>
+
+      <EditCard>
+        <:title>Mandaat</:title>
+        <:card as |Card|>
+          <Card.Columns>
+            <:left as |Item|>
+              <Item @labelFor="board-position">
+                <:label>Bestuursfunctie</:label>
+                <:content>
+                  <MandateRoleSelect
+                    @selected={{@model.mandate.roleBoard}}
+                    @onChange={{fn (mut @model.mandate.roleBoard)}}
+                  />
+                </:content>
+              </Item>
+              {{!-- TODO: Implement this once we know how it should work --}}
+              {{!-- <Item @labelFor="">
```

Comment:
The user will know how to use it. You only need to show this when selecting a specific Lid position in the Bestuursfunctie.

Reply:
Yea, which one though. That's why I hid it for now :smile:.

Reply:
The ID is 2e021095727b2464459a63e16ebeafd

Reply:
or this one here 5972fccd87f864c4ec06bfbd20b5008b

Reply:
Done done done.

Reply:
🚀